diff --git a/node_modules/ipfs-utils/src/fetch.js b/node_modules/ipfs-utils/src/fetch.js
index a941d92..4ab298d 100644
--- a/node_modules/ipfs-utils/src/fetch.js
+++ b/node_modules/ipfs-utils/src/fetch.js
@@ -15,4 +15,4 @@ if (isElectronMain) {
   impl = 'electron-fetch'
 }
 
-module.exports = require(impl)
+module.exports = require('native-fetch')
diff --git a/node_modules/ipfs-utils/src/http/fetch.js b/node_modules/ipfs-utils/src/http/fetch.js
index 1a27129..e5df9bc 100644
--- a/node_modules/ipfs-utils/src/http/fetch.js
+++ b/node_modules/ipfs-utils/src/http/fetch.js
@@ -17,6 +17,6 @@ if (typeof XMLHttpRequest === 'function') {
 }
 
 /** @type {fetchImpl} */
-const fetch = require(implName)
+const fetch = require('./fetch.node')
 
 module.exports = fetch
